mathesar_ui/src/systems/table-view/constraints/ConstraintCollapseHeader.svelte

```diff
@@ -53,16 +35,16 @@
   .column-names {
     display: flex;
     flex-direction: row;
-
-    > :global(* + *) {
-      margin-left: 0.25rem;
-    }
+    -webkit-flex-wrap: wrap;
+    flex-wrap: wrap;
+    -ms-flex-wrap: wrap;
```

Comment:
`flex-wrap` seems pretty [supported without a vendor prefix](https://caniuse.com/mdn-css_properties_flex-wrap). I think its safe to remove them.

Reply:
Removed and updated